docarray/array/storage/annlite/getsetdel.py

```diff
@@ -42,15 +42,15 @@ def _set_docs_by_ids(self, ids, docs: Iterable['Document'], mismatch_ids: Dict):
     def _del_docs_by_ids(self, ids):
         self._annlite.delete(ids)
 
+    def sync(self) -> None:
```

Comment:
if u do not need to override, no need to add it here

Reply:
Oh I hadnt realized this was all that was left here 😅 good catch